src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import "./Navbar.scss"
+import Logo from assets/skillChain.svg 
```

Comment:
Please add quotation marks around the asset Path and use camelCase for asset-import-name, e.g. `logo`

Reply:
and add './assets' to front

Reply:
and the svg file is wrong it contains only text